test/unicode/utf32.jl

```diff
@@ -171,3 +171,34 @@ u8 = "\U10ffff\U1d565\U1d7f6\U00066\U2008a"
 w = wstring(u8)
 @test length(w) == 5 && utf8(w) == u8 && collect(u8) == collect(w)
 @test u8 == WString(w.data)
+
+# Coverage improvements
```

Comment:
While true, we would hope that anything in this file is a coverage improvement. Can probably leave this comment out.